common/build_data_linkstamp.h

```diff
@@ -0,0 +1,25 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_COMMON_BUILD_DATA_LINKSTAMP_H_
+#define CARBON_COMMON_BUILD_DATA_LINKSTAMP_H_
+
+namespace Carbon::BuildData::Internal {
+
+// See build_data.h; the list of names here should match.
+//
+// These are exposed as non-constexpr so that `build_data_linkstamp.cpp` can be
+// compiled at the end, even though dependencies are added earlier.
```

Comment:
I wasn't sure at first what this comment was trying to say, but I think I figured it out -- would this edit make sense to you? Or did I misunderstand?

```suggestion
// These are exposed as non-constexpr as they need to be initialized 
// in `build_data_linkstamp.cpp` that is separately compiled.
```

Reply:
Elaborated on this comment.